crates/frontend/src/styles/vfs_item.rs

```diff
@@ -8,6 +8,8 @@ pub fn list(dom: DomBuilder<HtmlElement>) -> DomBuilder<HtmlElement> {
 
 pub fn body(dom: DomBuilder<HtmlElement>) -> DomBuilder<HtmlElement> {
     dom.apply(super::icon_text)
+        // this grid doesnt change it is just there to make the icon for file and the filename remain inline
+        // so we can leave it in styles
```

Comment:
Interesting, let's see how the code evolves. Perhaps what I said before about visual styles vs. layout styles was a bit too simplified.